docs/source/release_notes.rst

```diff
@@ -3,15 +3,17 @@
 Release Notes
 -------------
 
-.. Future Release
-  ==============
+Future Release
+===================
```

Comment:
You just have to match the length of the header. Doing more or less gives an error in docs build